code/modules/economy/account.dm

```diff
@@ -85,7 +88,16 @@
 				M.playsound_local(get_turf(M), 'sound/machines/twobeep.ogg', 50, TRUE)
 				if(M.can_hear())
 					to_chat(M, "[icon2html(A, M)] *[message]*")
-
+					
+/datum/bank_account/proc/log_money(amount,target)
+	if(!amount)
+		return
+	for(var/obj/A in bank_cards)
+		var/obj/item/pda/card_holder = recursive_loc_check(A, /obj/item/pda)
+		if(istype(card_holder,/obj/item/pda)) //check if it is inside pda
+			card_holder.brec += "<i><b>&larr; [amount] [amount > 0 ? "from " : " to "] [target] </b></i><br>"
```

Comment:
For now it is a simple message and only logs if the ID is inside a PDA when money goes in or out.

It shows like
```
-500 to 198791 Account
500 from 198791 Account
```

Should this get a timestamp? I fear that if I put too much information this wont be so nice